### PR TITLE
Logo fix - light and dark modes

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

**Review by salmad3 — COMMENTED** [review]
It works if we had "Polkadot Wiki" for the logo on the README since the logo and main header are like duplicates. Would also look better if it was transparent - should we use the pink token logo instead?

<p align="center">

</p>

**Comment by DrW3RK** [comment]
Up to you! Thanks. Please just make sure it looks good in both the modes and is centered

**Comment by DrW3RK** [comment]
Just committed the change. Please review, correct it if needed and merge. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<img align="center" src="https://user-images.githubusercontent.com/25497083/147838114-91029ff4-f900-4f30-823b-b038b2d1b9b9.png" width="650" height="180">
+<img align="center" src="https://user-images.githubusercontent.com/25497083/148661419-419ad5b3-1b9f-480a-b723-3f292616730c.png" width="100">
```
Comment (salmad3) [inline README.md]:
```suggestion
<img align="center" src="https://user-images.githubusercontent.com/25497083/148661419-419ad5b3-1b9f-480a-b723-3f292616730c.png" width="200">
```

**Review by salmad3 — APPROVED** [review]

**Pull request merged**